UnityProject/Assets/Scripts/Systems/StationObjectives/Objectives/ShipResources.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using System.Text;
+using Strings;
+using Systems.Cargo;
+
+namespace StationObjectives
+{
+	/// <summary>
+	/// An objective to ship items to nanotrasen
+	/// </summary>
+	[CreateAssetMenu(menuName = "ScriptableObjects/StationObjectives/ShipResources")]
+	public class ShipResources : StationObjective
+	{
+		/// <summary>
+		/// The pool of possible resources to ship
+		/// </summary>
+		[SerializeField]
+		private ItemDictionary ItemPool = null;
+
+		/// <summary>
+		/// The resource to ship
+		/// </summary>
+		private string ItemName;
+
+		/// <summary>
+		/// The number of items needed to ship to complete the objective
+		/// </summary>
+		private int Amount;
+
+		/// <summary>
+		/// Current amount of the item sold.
+		/// </summary>
+		private int AmountSold = 0;
+
+		protected override bool CheckCompletion()
+		{
+			var finalReport = new StringBuilder(victoryDescription);
+			finalReport.Replace("SHIPPEDVAL", $"{AmountSold}");
```

Comment:
use ``AppendFormat()`` instead.
```cs
finalReport.AppendFormat("{0}", AmountSold);
```

Reply:
actually in this case you don't need to use StringBuilder at all. Use it when you are concatenating multiple strings.